liqd_product/templates/header.html

```diff
@@ -40,6 +41,23 @@
                     {% userindicator %}
                 </div>
             </div>
+            <div class="main-header__lang">
+                {% get_available_languages as languages %}
+                {% for lang_code, lang_name in languages %}
+                    {% if lang_code != LANGUAGE_CODE %}
+                        {% get_language_info for lang_code as lang %}
+                            <form class="lang__switch" action="{% url 'set_language' %}" method="post">{% csrf_token %}
+                                <input name="next" type="hidden" value="{{ redirect_to }}" />
+                                <input name="language" type="hidden" value="{{ lang_code }}">
+                                {% if LANGUAGE_CODE == 'de' %}
+                                <a class="lang__btn--left" role="button">de</a><button class="lang__btn--right" type="submit">en</button>
+                                {% else %}
+                                <button class="lang__btn--left" type="submit">de</button><a class="lang__btn--right" role="button">en</a>
+                                {% endif %}
```

Comment:
this could maybe be done with styled radio buttons or links with data-toggling? but I could not get the behavior right and this seemed an easy and not too ugly compromise and a quick one, what do you think?

Reply:
Yes, that is fine (for now) and as long as we only have two languages. But if we add more, we have to change design anyway!

Reply:
cool, more languages will mean a dropdown and that works perfectly :D